VAMobile/src/store/slices/prescriptionSlice.ts

```diff
@@ -53,6 +54,8 @@ export type PrescriptionState = {
   refillRequestSummaryItems: RefillRequestSummaryItems
   tabCounts: TabCounts
   prescriptionsNeedLoad: boolean
+  prescriptionStatusCount: PrescriptionStatusCountData
+  prescriptionFirstRetrieval: boolean
```

Comment:
A small nit, but I think naming this `preloadComplete` would keep it consistent with the other slices

Reply:
I don't actually think 'preloadComplete' is a very good name for these and really we arent 'preloading' anything anymore and just full loading on login.